src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java

```diff
@@ -14,9 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.app.impl.config;
-
-import com.google.cloud.tools.app.api.Configuration;
+package com.google.cloud.tools.app.api;
```

Comment:
It does refer to appengine. The source of the abbreviation it is really the `gcloud preview app ...` command.